### PR TITLE
prepare blog post for public version

### PR DESCRIPTION
**Pull request by biphasic**

### CODE REVIEW

File: content/post/file-reading-benchmarking/index.md [hunk 1]
```diff
@@ -1,376 +1,31 @@
...
-            tsHigh = buff & mask_12b; 
-    }
-
-    return isEvent; 
-}
-```
-
-This code is just for demonstration purposes, it won't actually work, since we need to take care of other things such as timestamp overflows. A working version of this code is provided [here](https://github.com/open-neuromorphic/expelliarmus/blob/cc9fbf1f53bfccd75c920e37d4ed94aa5aec3b1b/expelliarmus/src/evt3.c#L256).
-
-## Authors
```
Comment (fabrizio-ottati) [inline content/post/file-reading-benchmarking/index.md:369]: Shouldn't we list ourselves as authors?

Comment (biphasic) [reply content/post/file-reading-benchmarking/index.md:369]: yes good point

Comment (fabrizio-ottati) [reply content/post/file-reading-benchmarking/index.md:369]: You can uncomment my part and add your description

Comment (biphasic) [reply content/post/file-reading-benchmarking/index.md:369]: I added back your section!

**Pull request merged**